lib/command/src/Obelisk/Command/VmBuilder.hs

```diff
@@ -98,12 +98,12 @@ setupNixDocker stateDir = withSpinner ("Creating Docker container named " <> con
   callProcessAndLogOutput (Debug, Error) $
     proc rmPath ["-f", stateDir </> sshKeyFileName, stateDir </> sshKeyFileName <.> "pub"]
   callProcessAndLogOutput (Debug, Error) $
-    proc "ssh-keygen" ["-t", "ed25519", "-f", stateDir </> sshKeyFileName, "-P", ""]
+    proc sshKeygenPath ["-t", "ed25519", "-f", stateDir </> sshKeyFileName, "-P", ""]
 
   -- Build the docker container (which uses the SSH keys in the 'ssh' folder)
   containerId <- fmap T.strip $ readProcessAndLogStderr Error $
-    proc "docker" ["build", stateDir, "--quiet"]
-  callProcessAndLogOutput (Debug, Error) $ proc "docker"
+    proc dockerPath ["build", stateDir, "--quiet"]
+  callProcessAndLogOutput (Debug, Error) $ proc dockerPath
```

Comment:
Let's leave docker out of this for now. None of these commands will work unless they have docker installed anyway and this is only for mac.

Reply:
docker has been left out of it.

Reply:
Excellent thanks.